generated/fig/tendermint/codec/v1/codec.proto

```diff
@@ -0,0 +1,802 @@
+syntax = "proto3";
+package tendermint;
+option  go_package = "github.com/figment-networks/tendermint-protobuf-def/pb/tendermint;pbcodec";
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/descriptor.proto";
+import "google/protobuf/any.proto";
+
+//------------------------------------------------------------------------
+// imported from cosmos/crypto/multisig/v1beta1/multisig.proto
+//
```

Comment:
Do we control the order in which these files are imported? I'd put the Tendermint types first.